test/kwtool/crawler/keyword_results_test.exs

```diff
@@ -0,0 +1,69 @@
+defmodule Kwtool.KeywordResultsTest do
+  use Kwtool.DataCase, async: true
+
+  alias Kwtool.Crawler.KeywordResults
+
+  describe "search/2" do
+    test "given an existing keyword, returns the matching keyword result" do
+      user = insert(:user)
+      keyword = insert(:keyword, phrase: "coffee", status: "finished", user: user)
+      insert(:keyword_result, keyword: keyword)
+
+      [keyword_result] = KeywordResults.search(user, %{keyword: "coffee"})
+
+      assert Enum.count(keyword_result.keyword_results) == 1
+      assert keyword.id == keyword_result.id
+      assert keyword.phrase == "coffee"
+    end
+
+    test "given an existing keyword with Url, returns the matching keyword and url" do
+      user = insert(:user)
+      keyword = insert(:keyword, phrase: "coffee", status: "finished", user: user)
+      insert(:keyword_result, keyword: keyword, organic_result_urls: ["https://star-coffee.com"])
```

Comment:
Should we add another keyword containing `coffee` but with different url.To test that it should not returned.

Reply:
Added